models/dashboard_group.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 // DashboardGroup dashboard group
+// Example: isResource
```

Comment:
Please add only those files related to the escalation chain.

Reply:
Done